ProjFS.Mac/PrjFSKext/VnodeCache.cpp

```diff
@@ -428,3 +505,30 @@ KEXT_STATIC bool TryInsertOrUpdateEntry_ExclusiveLocked(
     
     return false;
 }
+
+KEXT_STATIC_INLINE void InitCacheStats()
+{
+    atomic_exchange(&s_cacheStats.cacheEntries, 0U);
```

Comment:
Is this function called multi-threaded?  If not does it have to be an assignment?
Are there any perf impacts from using atomic_exchange?

Reply:
See related comment, I can update these to using `atomic_store_explicit` with `memory_order_relaxed` which should be a cheaper operation than `=`.